src/tls.c

```diff
@@ -388,21 +392,28 @@ int BuildTlsFinished(WOLFSSL* ssl, Hashes* hashes, const byte* sender)
 {
     int         ret;
     const byte* side;
-    byte        handshake_hash[HSHASH_SZ];
+    byte*       handshake_hash;
```

Comment:
I wouldn't have thought it too big to require allocating.
Could the code use WOLFSSL_SMALL_STACK?